frontend/apps/reader/modules/readerstyletweak.lua

```diff
@@ -783,13 +784,125 @@ local BOOK_TWEAK_INPUT_HINT = T([[
 %2]], _("You can add CSS snippets which will be applied only to this book."), BOOK_TWEAK_SAMPLE_CSS)
 
 local CSS_SUGGESTIONS = {
-    { "-cr-hint: footnote-inpage;", _("When set on a block element containing the target id of a href, this block element will be shown as an in-page footnote.")},
-    { "-cr-hint: non-linear-combining;", _("Can be set on some specific DocFragments (ie. DocFragment[id*=16]) to ignore them in the linear pages flow.")},
-    { "-cr-hint: toc-level1;", _("When set on an element, its text can be used to build the alternative table of contents.")},
-    { "display: run-in !important;", _("When set on a block element, this element content will be inlined with the next block element.")},
-    { "font-size: 1rem !important;", _("1rem will enforce your main font size.")},
-    { "hyphens: none !important;", _("Disables hyphenation inside the targeted elements.")},
-    { "text-indent: 1.2em !important;", _("1.2em is our default text indentation.")},
+    { _("Long-press for info ⓘ"), _([[
+This popup menu provides a quick CSS cheat sheet, showing common selector syntax and classic CSS properties.
+It also shows some KOReader specific, non-standard, CSS features, that can be useful with e-books.
```

Comment:
More a matter of taste, but I think it reads more fluid without the commas.
```suggestion
It also shows some KOReader-specific non-standard CSS features, that can be useful with e-books.
```